cluster/plan.go

```diff
@@ -602,6 +603,11 @@ func (c *Cluster) BuildKubeletProcess(host *hosts.Host, serviceOptions v3.Kubern
 		Env = append(Env,
 			// Enable running cri-dockerd
 			fmt.Sprintf("%s=%s", KubeletCRIDockerdNameEnv, "true"))
+		if c.multipleCIDRsConfigured() {
```

Comment:
What about using the library https://pkg.go.dev/k8s.io/utils/net with functions such as IsDualStackCIDRStrings. That will verify that the CIDRs include IPv4 and IPv6

Reply:
Nice, going to check this.